### PR TITLE
Running Live Tests

### PR DESCRIPTION
**Pull request by mohsha-msft**
- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by antkmsft** [comment]
To tell the CI run live tests against this PR, I think you need to put this comment: `/azp run go - azblob`

File: sdk/storage/azblob/zt_client_provided_key_test.go [hunk 1]
```diff
@@ -22,31 +25,37 @@ Azure Storage supports following operations support of sending customer-provided
...
-var testEncryptedKey = "MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc="
-var testEncryptedHash = "3QFFFpRA5+XANHqwwbT4yXDmrT/2JaLt/FKHjzhOdoE="
-var testEncryptionAlgorithm = "AES256"
-var testCPKByValue = CpkInfo{
-	EncryptionKey:       &testEncryptedKey,
-	EncryptionKeySHA256: &testEncryptedHash,
-	EncryptionAlgorithm: &testEncryptionAlgorithm,
+func getCPKEncryptionKey() *CpkInfo {
+	return &CpkInfo{
+		EncryptionKey:       to.StringPtr("MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc="),
```
Comment (seankane-msft) [inline sdk/storage/azblob/zt_client_provided_key_test.go:30]: Are these valid keys? if so, they should not be stored in plain text and should be read in via environment variables

Comment (richardpark-msft) [reply sdk/storage/azblob/zt_client_provided_key_test.go:30]: Looks like it's just "01234567012345670123456701234567". 

To make it easier on future reviews you might just want to base 64 encode that directly in your test, so it's obvious what it's coming from.

**Comment by mohsha-msft** [comment]
keep open.

**Comment by jhendrixMSFT** [comment]
Closing in favor of https://github.com/Azure/azure-sdk-for-go/pull/19211

**Pull request closed without merging**